fe/fe-core/src/main/java/org/apache/doris/nereids/operators/plans/logical/LogicalAggregation.java

```diff
@@ -42,18 +42,27 @@
 public class LogicalAggregation extends LogicalUnaryOperator {
 
     private final List<Expression> groupByExpressions;
-    private final List<? extends NamedExpression> outputExpressions;
+    private List<NamedExpression> outputExpressions;
```

Comment:
all operator should immutable, always generate new operator if u want to change attributes in it